.github/workflows/ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: Install stable toolchain
         uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: stable
+          toolchain: 1.79.0
```

Comment:
libfuzzing has a transient dependency to time 0.34 that does not work with Rust 1.80.
Retrograde to Rust 1.79.0, waiting for an upgrade.